common.yml

```diff
@@ -1691,10 +1702,15 @@ tasks:
             set -e
             go vet -composites=false ./bsondump ./mongo*
 
+  - name: check-third-party-notices
+    commands:
+      - func: "fetch source"
+      - func: "check third-party notices"
+
   - name: check-sbom-lite
     commands:
       - func: "fetch source"
-      - func: "check sbom lite"
+      - func: "check third-party notices"
```

Comment:
This change isn't quite right; this should still be "check sbom lite".

Reply:
Oops, thanks. I really wish Evergreen config had one _less_ level of structure (the variants -> tasks -> functions -> commands thing is one level too many).

Reply:
Fixed.